src/main/java/com/google/devtools/build/lib/analysis/test/ExecutionInfo.java

```diff
@@ -32,9 +32,11 @@ public final class ExecutionInfo extends NativeInfo implements ExecutionInfoApi
       new BuiltinProvider<ExecutionInfo>("ExecutionInfo", ExecutionInfo.class) {};
 
   private final ImmutableMap<String, String> executionInfo;
+  private final String execGroup;
 
-  public ExecutionInfo(Map<String, String> requirements) {
+  public ExecutionInfo(Map<String, String> requirements, String execGroup) {
```

Comment:
This caused a cyclic dependency, because ExecutionInfo can't refer back to BaseRuleClasses. I've moved the TEST_RUNNER_EXEC_GROUP constant into ExecutionInfo, with a slightly altered name. Please let me know whether that's acceptable.

Reply:
I'm not wild about this, but can't think of anything better. Can you add:

```
// TODO(bazel-team): Find a better location for this constant.
```